### DISCUSSION TITLE
Capacitor

### DISCUSSION THREAD

**Discussion by GraT3004** [termux/termux-app#3142] [General]
Hi, I want to know how to use capacitor for making an application on termux

**Comment by ChiefMikeK** [comment]
Start by reading the MANPAGE(S)

https://www.die.net/search/?q=capacitor&sa=Search&ie=ISO-8859-1&cx=partner-pub-5823754184406795%3A54htp1rtx5u&cof=FORID%3A9&siteurl=linux.die.net%2Fman%2F&ref=www.die.net%2F&ss=#gsc.tab=0&gsc.q=capacitor&gsc.page=1
